docker-compose.yml

```diff
@@ -33,6 +33,7 @@ services:
       python -m flask db upgrade & \
       python -m debugpy --listen 0.0.0.0:8081 -m flask run --host 0.0.0.0 --port 8080 --reload --debug --cert=/app/certs/cert.pem --key=/app/certs/key.pem
       "
+    env_file: .env
```

Comment:
If we are using .env can we add copying .env.example to the makefile, or the readme?